### DISCUSSION TITLE
[Question]:Manual installation on Windows

### DISCUSSION THREAD

**Discussion by maggieqnr** [danny-avila/LibreChat#1081] [Q&A] [answered]
### What is your question?

The Manual installation on Windows only have to install git and node? No need to install MongoDB？I saw the manual installation on Mac and Linux have to download MongoDB.

### More Details

![搜狗截图20231020171111](https://github.com/danny-avila/LibreChat/assets/146079448/7d7737b6-6d04-4311-9f85-05cab269cda9)
![搜狗截图20231020171154](https://github.com/danny-avila/LibreChat/assets/146079448/3602ff09-b53b-47f0-8c47-c0c70c0691d5)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
Please make your questions as discussions or we will keep moving them. MongoDB is needed in every installation unless you use MongoDB atlas
